package.json

```diff
@@ -33,10 +33,10 @@
     "conventional-changelog-conventionalcommits": "^5.0.0",
     "eslint": "^7.15.0",
     "gh-pages": "^3.1.0",
-    "gulp": "^4.0.2",
```

Comment:
Just wondering, are these changes to change the package version for `gulp` and `js-green-licenses` needed?

Reply:
@wich229 as for these change though, if you feel this was not something you meant to change, you could undo the changes. Again, I am not sure about that, just wanted to bring it up, since it seemed like a side-effect to me :)

Reply:
Thank you for sharing your thoughts and suggestions. I'll await the maintainer's feedback and make any necessary adjustments. :)

Reply:
Yes we would like to rever these changes. If they are necessary can you explain why? then we can get them upgraded as part of a separate change!